cms/djangoapps/contentstore/signals/apps.py

```diff
@@ -0,0 +1,22 @@
+"""
```

Comment:
@nasthagiri Yep, I just squashed my commits, but I was playing around with the settings. Was intending to commit apps.py at the top level. Thanks!